Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
AssertionError
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
AssertionError
AssertionError
AssertionError
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.11
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.17
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.7
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.13
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.5
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.16
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.9
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.8
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.10
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.4
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.12
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.14
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.18
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.15
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.2
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.19
srun: error: nid004548: task 0: Exited with exit code 1
srun: error: nid004548: task 0: Exited with exit code 1
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.3
srun: Terminating StepId=41587882.6
srun: Terminating StepId=41587882.1
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.0
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
AssertionError
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
Traceback (most recent call last):
Traceback (most recent call last):
           ^^^^^^^^^^^^^^^^^^
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
AssertionError
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
Traceback (most recent call last):
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.20
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.32
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.33
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.29
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.30
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.34
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.36
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.23
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.21
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.22
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.28
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.27
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.24
srun: error: nid004548: task 0: Exited with exit code 1
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.26
srun: Terminating StepId=41587882.31
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.25
Traceback (most recent call last):
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.35
srun: error: nid004548: task 0: Exited with exit code 1
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.37
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.39
srun: Terminating StepId=41587882.38
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
Traceback (most recent call last):
AssertionError
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
           ^^^^^^^^^^^^^^^^^^
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
AssertionError
Traceback (most recent call last):
AssertionError
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
Traceback (most recent call last):
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
Traceback (most recent call last):
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
AssertionError
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
AssertionError
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.41
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.53
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.51
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.40
srun: error: nid004548: task 0: Exited with exit code 1
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.64
srun: Terminating StepId=41587882.76
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
AssertionError
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
usage: calc_fc.py [-h] [--sub_id SUB_ID] [--ses_id SES_ID] [--task_id TASK_ID] [--atlas_spec ATLAS_SPEC]
                  [--n_rois N_ROIS] [--n_trs N_TRS] [--n_folds N_FOLDS] [--model MODEL] [--cv CV]
                  [--proc_type PROC_TYPE] [--profile] [--fc_data_path FC_DATA_PATH]
                  [--max_iter MAX_ITER]
calc_fc.py: error: argument --sub_id: expected one argument
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.57
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.60
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.67
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
AssertionError
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.45
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.87
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.54
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.50
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.66
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.52
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.55
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.65
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
Traceback (most recent call last):
           ^^^^^^^^^^^^^^^^^^
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
AssertionError
Traceback (most recent call last):
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
           ^^^^^^^^^^^^^^^^^^
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
           ^^^^^^^^^^^^^^^^^^
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
Traceback (most recent call last):
AssertionError
Traceback (most recent call last):
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
  File "/global/homes/m/mphagen/functional-connectivity/connectome-comparison/scripts/processing/calc_fc.py", line 131, in <module>
    assert len(ts_files) == 1
           ^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^
AssertionError
AssertionError
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.43
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.47
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.44
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.48
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.91
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.68
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.46
srun: error: nid004548: task 0: Exited with exit code 1
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.81
srun: Terminating StepId=41587882.42
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.69
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.74
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.78
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.49
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.80
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.98
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.100
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.70
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.62
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.99
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.59
srun: error: nid004548: task 0: Exited with exit code 2
srun: Terminating StepId=41587882.72
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.96
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.79
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.71
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.97
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.73
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.82
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.58
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.63
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.84
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.56
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.77
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.75
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.61
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.88
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.85
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.83
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.95
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.94
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.86
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.92
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.89
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.90
srun: error: nid004548: task 0: Exited with exit code 1
srun: Terminating StepId=41587882.93
